api/Window.json

```diff
@@ -2829,6 +2865,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "relative-multi-screen": {
+          "__compat": {
+            "description": "Moved relative to the <a href='https://developer.mozilla.org/docs/Web/API/Window_Management_API/Multi-screen_origin'>Multi-screen origin</a>",
+            "support": {
+              "chrome": {
+                "version_added": "100"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "1",
+                "notes": "Multi-screen coordinates are used, but windows are clamped to their existing displays."
```

Comment:
I might either omit this, or note similar clamping in Chrome without window-management permission, and ditto for Chrome's window.open().

Reply:
Notes added as requested:

> Without the <code>window-management</code> permission, multi-screen coordinates are used, but windows are clamped to their existing displays.